### PR TITLE
doc: wifi: update TWT

### PR DESCRIPTION
**Pull request by kittydepa**
Update TWT sample with Kconfig option needed to be able to run the traffic server

### CODE REVIEW

File: samples/wifi/twt/README.rst [hunk 1]
```diff
@@ -31,13 +31,17 @@ When the traffic generator module is used with this sample, the traffic generato
 The server source is available in the :file:`scripts/traffic_gen_server` file.
 This is a `python3` based server, so you must make sure that `python3` is installed on your PC.
 
-To start the server, run the following command:
+To start the server:
 
-.. code-block:: console
+1. Set the :kconfig:option:`CONFIG_TRAFFIC_GEN_REMOTE_IPV4_ADDR` Kconfig option to the IPv4 address of the host running the traffic generator server.
```
Comment (krish2718) [inline samples/wifi/twt/README.rst:36]: Should we clarify that this is for the client side i.e., for Zephyr build? or is it implied?

**Review by bama-nordic — APPROVED** [review]

File: samples/wifi/twt/README.rst (hunk 1 above)
Comment (richabp) [inline samples/wifi/twt/README.rst:36]:
```suggestion
1. Set the :ref:`CONFIG_TRAFFIC_GEN_REMOTE_IPV4_ADDR` Kconfig option to the IPv4 address of the host running the traffic generator server.
```

**Review by richabp — CHANGES_REQUESTED** [review]
Do we need  a changelog entry for this as this Kconfig option is already available in the sample, the new instruction is about setting it to the IPv4 address of the host.

**Review by richabp — APPROVED** [review]

File: samples/wifi/twt/README.rst (hunk 1 above)
Comment (ioannisg) [reply samples/wifi/twt/README.rst:36]: I think this is implied well @kittydepa

**Review by ioannisg — APPROVED** [review]

**Pull request merged**